pyndustri.pyi

```diff
@@ -169,69 +206,99 @@ class Senseable(ABC):
+        """The amount of surge_alloy stored or carried by the link"""
     @property
     def spore_pod(self) -> int:
-        """Amount of Spore Pod stored or carried by the link."""
-    @property
-    def sand(self) -> int:
-        """Amount of Sand stored or carried by the link."""
+        """The amount of spore_pod stored or carried by the link"""
     @property
     def blast_compound(self) -> int:
-        """Amount of Blast Compound stored or carried by the link."""
+        """The amount of blast_compound stored or carried by the link"""
     @property
     def pyratite(self) -> int:
-        """Amount of Pyratite stored or carried by the link."""
+        """The amount of pyratite stored or carried by the link"""
     @property
-    def metaglass(self) -> int:
-        """Amount of Metaglass stored or carried by the link."""
+    def beryllium(self) -> int:
+        """The amount of beryllium stored or carried by the link"""
     @property
-    def scrap(self) -> int:
-        """Amount of Scrap stored or carried by the link."""
+    def tungsten(self) -> int:
+        """The amount of tungsten stored or carried by the link"""
+    @property
+    def oxide(self) -> int:
+        """The amount of oxide stored or carried by the link"""
+    @property
+    def carbide(self) -> int:
+        """The amount of carbide stored or carried by the link"""
     @property
     def water(self) -> int:
-        """Amount of Water stored or carried by the link."""
+        """The amount of water stored or carried by the link"""
     @property
     def slag(self) -> int:
-        """Amount of Slag stored or carried by the link."""
+        """The amount of slag stored or carried by the link"""
     @property
     def oil(self) -> int:
-        """Amount of Oil stored or carried by the link."""
+        """The amount of oil stored or carried by the link"""
     @property
     def cryofluid(self) -> int:
-        """Amount of Cryofluid stored or carried by the link."""
+        """The amount of cryofluid stored or carried by the link"""
+    @property
+    def neoplasm(self) -> int:
+        """The amount of neoplasm stored or carried by the link"""
+    @property
+    def arkycite(self) -> int:
+        """The amount of arkycite stored or carried by the link"""
+    @property
+    def ozone(self) -> int:
+        """The amount of ozone stored or carried by the link"""
+    @property
+    def hydrogen(self) -> int:
+        """The amount of hydrogen stored or carried by the link"""
+    @property
+    def nitrogen(self) -> int:
+        """The amount of nitrogen stored or carried by the link"""
+    @property
+    def cyanogen(self) -> int:
+        """The amount of cyanogen stored or carried by the link"""
     @property
     def items(self) -> int:
         """The sum of all items contained or carried by the link."""
     @property
-    def first_item(self) -> Optional[Link]:
+    def first_item(self) -> Link | None:
```

Comment:
Curious why?

Reply:
I think it looks cleaner. Since Python3.11 (maybe 3.12) that's the "preferred" way of doing it I think. It's purely preference.